test/Directory.Build.props

```diff
@@ -37,14 +37,13 @@
     <Using Include="NSubstitute"/>
     <Using Include="NSubstitute.Core"/>
     <Using Include="NSubstitute.ReturnsExtensions"/>
-    <Using Include="VerifyXunit"/>
     <PackageReference Include="coverlet.msbuild" Version="3.1.0" />
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.0.0" />
-    <PackageReference Include="NSubstitute" Version="4.2.2" />
-    <PackageReference Include="FluentAssertions" Version="6.2.0" />
+    <PackageReference Include="NSubstitute" Version="4.3.0" />
+    <PackageReference Include="FluentAssertions" Version="6.4.0" />
     <PackageReference Include="xunit" Version="2.4.1" />
     <PackageReference Include="xunit.runner.visualstudio" Version="2.4.3" />
-    <PackageReference Include="Verify.Xunit" Version="14.11.1" />
+    <PackageReference Include="Verify.Xunit" Version="15.1.0" />
     <PackageReference Include="PublicApiGenerator" Version="10.2.0" />
```

Comment:
Do we even use his these days?

Reply:
we use it for the api approvals. and i intend to use it for the sql listener integration test